README.md

```diff
@@ -1,3 +1,8 @@
+[![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/pygsuite/pygsuite/LICENSE)
+[![PyPI](https://img.shields.io/pypi/v/pygsuite.svg)](https://pypi.org/project/pygsuite/)
+[![Python Versions](https://img.shields.io/pypi/pyversions/pygsuite.svg)](https://pypi.python.org/pypi/pygsuite)
+[![Downloads](https://img.shields.io/badge/dynamic/json.svg?label=downloads&url=https%3A%2F%2Fpypistats.org%2Fapi%2Fpackages%2Fpygsuite%2Frecent&query=data.last_month&colorB=brightgreen&suffix=%2FMonth)](https://pypistats.org/packages/pygsuite)
```

Comment:
Do we want to move these new badges down below the title next to the existing `gitter` badge?